examples/platform/silabs/SiWx917/FreeRTOSConfig.h

```diff
@@ -122,21 +122,21 @@ extern "C" {
 extern uint32_t SystemCoreClock;
 
 /* Energy saving modes. */
-#if defined(SL_CATALOG_POWER_MANAGER_PRESENT)
+#if defined(CHIP_DEVICE_CONFIG_ENABLE_SED)
```

Comment:
The term SED is a thread concept. I don't know if it should be used for Matter ICD in general
@mkardous-silabs  ^

Also, on EFR32 end the power manager can be used and as some effect even if you aren't a SED or ICD.
Hence the use of `SL_CATALOG_POWER_MANAGER_PRESENT` maybe this should be kept?